site/content/en/docs/start/_index.md

```diff
@@ -508,6 +508,8 @@ minikube start
 
 If minikube fails to start, see the [drivers page]({{<ref "/docs/drivers">}}) for help setting up a compatible container or virtual-machine manager.
 
+If minikube starts too slowly because of network problem(firewall etc), you can exec it with `--image-repository` flag.
```

Comment:
I dont thin this is the right place to suggest this feature. on the first page, this is not a common problem.
how about adding it it to FAQ
https://minikube.sigs.k8s.io/docs/faq/

or how about adding a new entry in proxy and VPN page

https://minikube.sigs.k8s.io/docs/handbook/vpn_and_proxy/

Reply:
i add an entry in faq